app/models/sentence.php

```diff
@@ -382,7 +382,7 @@ public function getRandomId($lang = 'und')
     public function getSeveralRandomIds($lang = 'und',  $numberOfIdWanted = 10)
     {
         if(Configure::read('Search.enabled') == false) {
-            return array(1);
+            return false;
```

Comment:
It would make more sense to return `null` rather than `false`, I think. Then everywhere where you used `is_bool()`, you can check instead with `is_null()`.